vacation-calendar/README.md

```diff
@@ -9,22 +9,37 @@ required.
 
 <!-- [START setup] -->
 
+First, setup Google Calendar:
+
+1. Open [Google Calendar][calendar] in your browser.
+1. [Create a new calendar][calendar_setup] called "Team Vacations".
+1. Still in the settings screen, select the new calendar from the left menu.
+1. Scroll to **Integrate calendar** and copy the value under **Calendar ID**.
+
+Next, create the Apps Script project:
+
 1. Open the [Apps Script project][project] in your browser.
 1. Click the menu item **File > Make a copy...**.
-1. Set the variable `TEAM_CALENDAR_ID` to the ID of a Google Calendar you have
-   edit access to.
+1. Set the variable `TEAM_CALENDAR_ID` to the ID of the calendar you
+   created earlier.
 1. Set the variable `GROUP_EMAIL` to the email address of a Google Group
    containing your team members.
+
+Finally, run the script:
```

Comment:
Drop "Finally,". You have further instructions after this, so not really the last thing to do :)